README.md

```diff
@@ -31,6 +31,7 @@ as ``ovirt_engine_setup_answer_file_path``.
 | ovirt_engine_setup_product_type       | oVirt                 | One of ["oVirt", "RHV"], case insensitive. |
 | ovirt_engine_setup_package_list       | []                    | List of extra packages to be installed on engine apart from ovirt-engine package. |
 | ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
```

Comment:
@pkubica Was it an intention that `ovirt_engine_setup_answer_file_path` is on localhost? If yes, we need to document it to make it clear and merge this patch.

Reply:
I'm note sure what was the intention was, but the template module taking the 'src:' path from localhost. In any case, in order to support using the answer file from a remote machine as well as localhost, we need to use another task like I added in the patch.